learnosity_sdk/request/dataapi.py

```diff
@@ -7,8 +6,8 @@
 
 
 class DataApi(object):
-
-    def request(self, endpoint, security_packet,
+    @staticmethod
```

Comment:
Is there a particular reason for changing this to a static method? I know the method doesn't reference `self`, but it might be best to keep it consistent with previous behaviour.

Also cautious about making changes we might regret later because _removing_ the static method in a future version would be a breaking change.

Reply:
Good point with regards to keeping it consistent. I shall change it back to the previous behaviour.